apps/api-gateway/src/verification/dto/request-proof.dto.ts

```diff
@@ -260,20 +257,44 @@ export class RequestProofDto extends ProofPayload {
     @IsOptional()
     @IsString({ message: 'comment must be in string' })
     comment: string;
-
+    
     type:ProofRequestType;
-
+    
     orgId: string;
-
-    @ApiPropertyOptional()
+    
+    @ApiPropertyOptional({enum:AutoAccept})
     @IsString({ message: 'auto accept proof must be in string' })
     @IsNotEmpty({ message: 'please provide valid auto accept proof' })
     @IsOptional()
     @IsEnum(AutoAccept, {
-        message: `Invalid auto accept proof. It should be one of: ${Object.values(AutoAccept).join(', ')}`
+      message: `Invalid auto accept proof. It should be one of: ${Object.values(AutoAccept).join(', ')}`
     })
     autoAcceptProof: AutoAccept;
+    version: string;
 }
+export class RequestProofDto extends RequestProofDtoBase {
```

Comment:
Maybe we can change the Dto name to something more meaningful since the only difference between the `RequestProofDto` and `ProofRequestPayloadDto` is the `connectionId` type(string and array respectively).

Reply:
Yes, Sure we can.